lib/src/css_parser.dart

```diff
@@ -12,6 +15,19 @@ Style declarationsToStyle(Map<String, List<css.Expression>> declarations) {
       case 'background-color':
         style.backgroundColor = ExpressionMapping.expressionToColor(value.first);
         break;
+      case 'border':
+        List<css.LiteralTerm> borderWidths = value.whereType<css.LiteralTerm>().toList();
+        /// List<css.LiteralTerm> might include other values than the ones we want for [BorderSide.width], so make sure to remove those before passing it to [ExpressionMapping]
+        borderWidths.removeWhere((element) => element.text != "thin" && element.text != "medium" && element.text != "thick" && !(element is css.LengthTerm));
+        List<css.Expression> borderColors = value.where((element) => ExpressionMapping.expressionToColor(element) != null).toList();
+        List<css.LiteralTerm> temp = value.whereType<css.LiteralTerm>().toList();
```

Comment:
Not a huge of generic variable names such as `temp`. But I guess it works fine.

Reply:
I agree, I copy/pasted off of the text-decoration code and it had temp. Changed it :)